lib/modules/manager/cargo/extract.ts

```diff
@@ -11,6 +11,16 @@ import type {
   CargoSection,
 } from './types';
 
+function get_cargo_index_env(registryName: string): string | null {
+  return (
+    process.env[
+      `CARGO_REGISTRIES_${registryName
+        .toUpperCase()
+        .replaceAll('-', '_')}_INDEX`
+    ] ?? null
+  );
+}
```

Comment:
```suggestion
function getCargoIndexEnv(registryName: string): string | null {
  const registry = registryName.toUpperCase().replaceAll('-', '_');
  return process.env[`CARGO_REGISTRIES_${registry}_INDEX`] ?? null;
}
```
Use camelCase and improve readability.